app-text/asciidoc/asciidoc-8.6.10_p20181016.ebuild

```diff
@@ -15,8 +15,7 @@ KEYWORDS="alpha amd64 arm arm64 hppa ia64 m68k ~mips ppc ppc64 ~riscv s390 sh sp
 
 LICENSE="GPL-2"
 SLOT="0"
-IUSE="examples graphviz highlight test"
-RESTRICT="!test? ( test )"
```

Comment:
You can just add `RESTRICT="test"` and put a short comment on top of it explaining why it's disabled. I'd much prefer that over to removing these pieces of information what was/is needed for test phase.

Reply:
What pieces of information are you referring to that would be missing? The same bits of information are in the new 9.0.0_rc1 ebuild.  But whichever you prefer, I can also just not touch the test functionality, but then I have to add the dvisvgm dependency like I did for 9.0.0_rc1.

Reply:
I don't think touching this ebuild at all is very relevant anymore, doubt you will get bug reports for it. It's good if they are fixed in a later version.

Reply:
OK